institutions-api/db/db.py

```diff
@@ -47,15 +52,38 @@ def get_institution(short_id: str) -> InstitutionModel:
 
         return InstitutionModel.from_institution(institution)
 
+@sqlalchemy_http_exceptions
 def add_institution(institution: InstitutionModel, author: OIDCUserInfo):
     """ Create a new institution """
     with DbSession() as session:
         inst = Institution(institution.name, institution.id, author.id)
-        if institution.ror_id:
-            inst.identifiers = [InstitutionIdentifier(_ror_id_type(session), institution.ror_id)]
         session.add(inst)
+        if institution.ror_id:
+            ror_id = InstitutionIdentifier(_ror_id_type(session), institution.ror_id, inst.id)
+            session.add(ror_id)
+
         session.commit()
 
+def _update_institution_ror_id(session: Session, institution: Institution, ror_id: str):
```

Comment:
@brianhlin re: the discussion in https://github.com/osg-htc/topology-institutions-api/pull/1 about not always falling through to the delete-then-recreate workflow, I've split it out here into 3 explicit cases for delete, create and update

Reply:
LGTM!